ACCESSIBILITY.md

```diff
@@ -0,0 +1,97 @@
+- Age
+- Disability
+- Ethnicity and race
+- Family
+- Language (written and/or spoken)
+- Geography
+- Nationality
+- Religion, faith and belief
+- Sex and gender
+- Sexual orientation
+- Socioeconomic status
+- Technical or disciplinary background
+
+We recognise the complex and interlinking ways in which intersectionality affects identity, and thereby, access or inclusion within any community project like _The Turing Way_. So in this sense, our accessibility practices may overlap with practices that enable inclusion more broadly within any community.
+
+In developing this policy, we aim to draw from the [access-centered framework](https://accesscenteredmovement.com/what-access-centered-means/) in describing access as a constant process that changes in each space with each individual. Therefore, while this policy outlines our commitment to access-centered practices within our community in a broad sense, we also document the specific practices we use within _The Turing Way_ our Community Handbook, as well as share broader recommendations that go beyond our current practices in the Guide for Accessibility. Each resource (our Accessibility Policy - this document, Community Handbook Accessibility Guidelines, and our Accessibility Guide) play a different yet overlapping role in enabling, documenting, and sharing our evolving language around accessible and inclusive collaboration.
+
+This policy is a work in progress, as are the practices that we employ as a community. Much like how _The Turing Way_ itself is always being written and rewritten, our accessibility policies and practices are constantly being updated as broader awareness grows and changes in technology are made.
+
+## 1.1 Core Principles
+
+These core principles are at the heart of our accessibility policy, and they extend beyond the web accessibility of our open access guides (see 2.1) to the broader practices that that we use as a community on collective platforms (see 2.2) and while hosting events (see 2.3).
+
+This means that questions of access are tied to how people read the guides as well as their ability to participate in the project and the community more broadly. Participation in _The Turing Way_ itself can be defined broadly, but doing so often requires navigating a number of different platforms. In all contexts, we aim to facilitate access in a broad sense through:
+
+1. **Removing barriers**: We actively advocate for and employ tactics that seek to remove barriers for any current or future participants in our community. This may include people who historically have been limited or structurely excluded from participating in through educational, geographical, linguistic, or technical barriers.
```

Comment:
```suggestion
1. **Removing barriers**: We actively advocate for and employ tactics that seek to remove barriers for any current or future participants in our community. This may include people who historically have been limited or structurally excluded from participating through educational, geographical, linguistic, or technical barriers.
```